corehq/ex-submodules/couchforms/models/__init__.py

```diff
@@ -0,0 +1,6 @@
+from django.conf import settings
```

Comment:
Definitely need all models available. We'll have to roll this out per domain at some point to going to need both to co-exist.